### DISCUSSION TITLE
system-wide installation of an appimage?

### DISCUSSION THREAD

**Discussion by Ptrolsen** [TheAssassin/AppImageLauncher#569] [Q&A] [answered]
The description of the option to move an appimage into  a "central location" in the opening dialogue is somewhat confusing. The default "central location" is in the user's home directory  tree. 
But can I choose a path in the root in order to have a shared copy of the appimage for all users on the system? To me, that would be a much more "central" location than the default, but it would of course require sudo root rights for the user.
I have not dared to test if such a path could actually be chosen, as it appears to me that  the documentation does not mention the possibility.

**Comment by maggu2810** [comment]
I am not using AppImageLauncher anymore but IMHO you can give whatever directory you want. For sure the user running AppImageLauncher requires write permissions for the directory to move files into it.

**Comment by TheAssassin** [comment, marked as answer]
There is some support for a `/Applications` directory that is shared by all users. `appimagelauncherd` watches this directory automatically and integrates AppImages discovered there per user.

If you prefer another directory, you can add it to `appimagelauncherd`'s watched directories in the settings. Please note however that this needs to be done for every user then.
